### DISCUSSION TITLE
Use icons or better colors to better differentiate points in Observation map

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-desktop#575] [Ideas]
**Describe the idea**
It would be nice to use icons or a better color scheme to better differentiate points.

**What's your role? Where and how do you currently use the application?**
Dd tech

**What is your ideal step by step workflow?**
Perhaps the way to represent points could be a setting.

**Have you thought about what it might look like?**
The icons could be the same as the ones defined in the mapeosettings file.

**Additional context**
Requested by NGO partners

**Comment by rudokemper** [comment]
This was commented upon or requested in several recent meetings with CIPDP (Kenya), Ohneganos (Canada), and Santo Sunset Environmental Network (Vanuatu).
